docs/integrations/sources/amazon-seller-partner.md

```diff
@@ -0,0 +1,51 @@
+# Amazon Seller Partner
+
+## Sync overview
+
+This source can sync data for the [Amazon Seller Partner API](https://github.com/amzn/selling-partner-api-docs/blob/main/guides/en-US/developer-guide/SellingPartnerApiDeveloperGuide.md).
+
+### Output schema
+
+This source is capable of syncing the following streams:
+
+* [Orders](https://github.com/amzn/selling-partner-api-docs/blob/main/references/orders-api/ordersV0.md)
+
```

Comment:
also it outputs an orders report. It's name is `GET_FLAT_FILE_ALL_ORDERS_DATA_BY_ORDER_DATE_GENERAL`

This is an unfortunate name. But it is the name.

Reply:
Done.